FirebaseAdmin/FirebaseAdmin/Auth/Multitenancy/TenantAwareFirebaseAuth.cs

```diff
@@ -38,6 +41,26 @@ internal TenantAwareFirebaseAuth(Args args)
         /// </summary>
         public string TenantId { get; }
 
+        /// <summary>
+        /// Creates a new Firebase session cookie from the given ID token and options. The returned JWT can
+        /// be set as a server-side session cookie with a custom cookie policy.
+        /// </summary>
+        /// <exception cref="FirebaseAuthException">If an error occurs while creating the cookie.</exception>
+        /// <param name="idToken">The Firebase ID token to exchange for a session cookie.</param>
+        /// <param name="options">Additional options required to create the cookie.</param>
+        /// <param name="cancellationToken">A cancellation token to monitor the asynchronous
+        /// operation.</param>
+        /// <returns>A task that completes with the Firebase session cookie.</returns>
```

Comment:
Can you use [inheritdoc](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/inheritdoc) here?

Reply:
Thanks. Good call.